specs/electra/beacon-chain.md

```diff
@@ -586,7 +586,8 @@ def get_consolidation_churn_limit(state: BeaconState) -> Gwei:
 ```python
 def get_pending_balance_to_withdraw(state: BeaconState, validator_index: ValidatorIndex) -> Gwei:
     return sum(
-        withdrawal.amount for withdrawal in state.pending_partial_withdrawals if withdrawal.index == validator_index
+        withdrawal.amount for withdrawal in state.pending_partial_withdrawals if
+        withdrawal.validator_index == validator_index
```

Comment:
Let's put the `if` on the 2nd line. I believe this is the most common style. For example:

https://github.com/ethereum/consensus-specs/blob/31cd9cbec2d93e5a6df84dcee9ca9e8aa77803ee/specs/electra/beacon-chain.md?plain=1#L607-L610

https://github.com/ethereum/consensus-specs/blob/31cd9cbec2d93e5a6df84dcee9ca9e8aa77803ee/specs/electra/beacon-chain.md?plain=1#L522-L526